doc/api/crypto.md

```diff
@@ -249,24 +249,26 @@ added: v1.0.0
   - `plaintextLength` {number}
 * Returns: {Cipher} for method chaining.
 
-When using an authenticated encryption mode (`GCM`, `CCM` and `OCB` are
-currently supported), the `cipher.setAAD()` method sets the value used for the
-_additional authenticated data_ (AAD) input parameter.
+When using an authenticated encryption mode (`GCM`, `CCM`, `OCB` and
```

Comment:
This is really calling out for a table of cipher names and capabilities somewhere - but that's not a problem this PR needs to address.

Reply:
I kept the original doc, and maybe another issue to do this